web/src/components/pages/contribution/listen/listen.tsx

```diff
@@ -140,6 +148,8 @@ class ListenPage extends React.Component<Props, State> {
         (clip, i) => (i === clipIndex ? { ...clip, isValid } : clip)
       ),
     });
+
+    updateKiosk({ listenFinished: true });
```

Comment:
Why set this here and in line 189?

Reply:
Because that's the only proper place I could find: when we vote, or when we have no clip to show. Happy to do that at some other, unique place for both case if you see where :)